cmd/kubeadm/app/cmd/upgrade/testdata/diff_controlplane_config.yaml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
hm, what type of linter is that?
files with a single kind should no have a leading `---`.
/hold

Reply:
I'm using https://pypi.org/project/yamllint/ :)